src/Microsoft.DotNet.Core.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -60,7 +58,7 @@ private void WriteRuntimeConfig()
             AddRuntimeOptions(config.RuntimeOptions);
 
             var runtimeConfigJsonFile =
-                Path.Combine(RuntimeOutputPath, OutputName + FileNameSuffixes.RuntimeConfigJson);
+                Path.Combine(RuntimeOutputPath, AssemblyName + ".runtimeconfig.json");
```

Comment:
I have actually been meaning to move these out to the MSBuild .targets file.  Moving now.